process_variance:
  [0.01, 0.01, 0.01, 0.05, 0.05, 0.05, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]

observation_variance: [0.1, 0.1, 0.1, 0.5, 0.5, 0.5]
